### DISCUSSION TITLE
Core 1 panic'ed in streams-i2s-i2s-2 example

### DISCUSSION THREAD

**Discussion by DrNoobBanana** [pschatzmann/arduino-audio-tools#1640] [Q&A] [answered]
thank you for your great library

I have a PCM1808 ADC and a PCM5102 DAC and an ESP32 WROOM 32. I connected everything as was suggested in the READ ME file. and when I upload the sketch, it just keeps printing this:

[I] Streamstarting I2S...
[I] AudioTypes.h : 127 - out: sample_rate: 44100 / channels: 2 / bits_per_sample: 16
[I] AudioTypes.h : 127 -  sample_rate: 44100 / channels: 2 / bits_per_sample: 16
[I] I2SConfigESP32V1.h : 66 - rx/tx mode: RX_MODE
[I] I2SConfigESP32V1.h : 67 - port_no: 0
[I] I2SConfigESP32V1.h : 68 - is_master: Master
[I] I2SConfigESP32V1.h : 69 - sample rate: 44100
[I] I2SConfigESP32V1.h : 70 - bits per sample: 16
[I] I2SConfigESP32V1.h : 71 - number of channels: 2
[I] I2SConfigESP32V1.h : 72 - signal_type: Digital
[I] I2SConfigESP32V1.h : 74 - i2s_format: I2S_STD_FORMAT
[I] I2SConfigESP32V1.h : 77 - use_apll: true
[I] I2SConfigESP32V1.h : 79 - buffer_count:6
[I] I2SConfigESP32V1.h : 80 - buffer_size:512
[I] I2SConfigESP32V1.h : 85 - pin_bck: 14
[I] I2SConfigESP32V1.h : 87 - pin_ws: 15
[I] I2SConfigESP32V1.h : 89 - pin_data: 22
[I] I2SConfigESP32V1.h : 91 - pin_data_rx: -1515870811
[I] I2SESP32V1.h : 197 - tx: -1, rx: -1515870811
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x400daede  PS      : 0x00060a30  A0      : 0x800dbb49  A1      : 0x3ffb1f90  
A2      : 0xa5a5a5a5  A3      : 0x0000009b  A4      : 0x00000001  A5      : 0x00000000  
A6      : 0xb33fffff  A7      : 0xb33fffff  A8      : 0xd5d70b4c  A9      : 0x3ffb1f90  
A10     : 0x3ffb8b84  A11     : 0x0000003c  A12     : 0x00060820  A13     : 0x3f409668  
A14     : 0xb33fffff  A15     : 0xb33fffff  SAR     : 0x0000000f  EXCCAUSE: 0x0000001c  
EXCVADDR: 0xd5d70b4c  LBEG    : 0x40089d84  LEND    : 0x40089d8f  LCOUNT  : 0x00000000  

Backtrace: 0x400daedb:0x3ffb1f90 0x400dbb46:0x3ffb1fb0 0x400dbe9b:0x3ffb1ff0 0x400d36b8:0x3ffb2020 0x400d3bf1:0x3ffb2080 0x400d3cd5:0x3ffb20b0 0x400d3ed9:0x3ffb2110 0x400d3f94:0x3ffb2170 0x400d9657:0x3ffb2270 0x4009067e:0x3ffb2290

ELF file SHA256: 5af6c45241b31cc4

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)

ELF file SHA256: 5af6c45241b31cc4

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1448
load:0x40078000,len:14844
ho 0 tail 12 room 4
load:0x40080400,len:4
load:0x40080404,len:3356
entry 0x4008059c
starting I2S...
[I] AudioTypes.h : 127 - out: sample_rate: 44100 / channels: 2 / bits_per_sample: 16
[I] AudioTypes.h : 127 -  sample_rate: 44100 / channels: 2 / bits_per_sample: 16
[I] I2SConfigESP32V1.h : 66 - rx/tx mode: RX_MODE
[I] I2SConfigESP32V1.h : 67 - port_no: 0
[I] I2SConfigESP32V1.h : 68 - is_master: Master
[I] I2SConfigESP32V1.h : 69 - sample rate: 44100
[I] I2SConfigESP32V1.h : 70 - bits per sample: 16
[I] I2SConfigESP32V1.h : 71 - number of channels: 2
[I] I2SConfigESP32V1.h : 72 - signal_type: Digital
[I] I2SConfigESP32V1.h : 74 - i2s_format: I2S_STD_FORMAT
[I] I2SConfigESP32V1.h : 77 - use_apll: true
[I] I2SConfigESP32V1.h : 79 - buffer_count:6
[I] I2SConfigESP32V1.h : 80 - buffer_size:512
[I] I2SConfigESP32V1.h : 85 - pin_bck: 14
[I] I2SConfigESP32V1.h : 87 - pin_ws: 15
[I] I2SConfigESP32V1.h : 89 - pin_data: 22
[I] I2SConfigESP32V1.h : 91 - pin_data_rx: -1515870811
[I] I2SESP32V1.h : 197 - tx: -1, rx: -1515870811
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x400daede  PS      : 0x00060a30  A0      : 0x800dbb49  A1      : 0x3ffb1f90  
A2      : 0xa5a5a5a5  A3      : 0x0000009b  A4      : 0x00000001  A5      : 0x00000000  
A6      : 0xb33fffff  A7      : 0xb33fffff  A8      : 0xd5d70b4c  A9      : 0x3ffb1f90  
A10     : 0x3ffb8b84  A11     : 0x0000003c  A12     : 0x00060820  A13     : 0x3f409668  
A14     : 0xb33fffff  A15     : 0xb33fffff  SAR     : 0x0000000f  EXCCAUSE: 0x0000001c  
EXCVADDR: 0xd5d70b4c  LBEG    : 0x40089d84  LEND    : 0x40089d8f  LCOUNT  : 0x00000000  
Backtrace: 0x400daedb:0x3ffb1f90 0x400dbb46:0x3ffb1fb0 0x400dbe9b:0x3ffb1ff0 0x400d36b8:0x3ffb2020 0x400d3bf1:0x3ffb2080 0x400d3cd5:0x3ffb20b0 0x400d3ed9:0x3ffb2110 0x400d3f94:0x3ffb2170 0x400d9657:0x3ffb2270 0x4009067e:0x3ffb2290

ELF file SHA256: 5af6c45241b31cc4

Rebooting...
ets Jul 29 2019 12:21:46

I also have a cheap analog microphone which is connected the left input of my ADC. 
the streams-i2s-serial example worked fine with my ADC, which requires an MCLK (SCK) input. my DAC also has a SCK pin but IDK where to connect it since it is not mentioned anywhere in the example.

**Comment by pschatzmann** [comment, marked as answer]
Make sure that you use an actual version of the core and all libraries and if you then still get a Backtrace you need to analyze it yourself to identify the the code line that is leading to the crash

Apart from this follow the [advice given in the Wiki](https://github.com/pschatzmann/arduino-audio-tools/wiki/It's-not-working)!
